gensim/models/word2vec.py

```diff
@@ -761,16 +761,23 @@ def _raw_word_count(self, job):
         """Return the number of words in a given job."""
         return sum(len(sentence) for sentence in job)
 
-    def train(self, sentences, total_words=None, word_count=0,
-              total_examples=None, queue_factor=2, report_delay=1.0):
+    def train(self, sentences, total_examples=None, total_words=None,
+              epochs=None, start_alpha=None, end_alpha=None,
```

Comment:
Ah, this is function definition, vertical is probably OK here. I misread it as function call when reviewing.

Hanging indent would be arguments on separate lines, with extra indentation.